Test/DemoSuite.js

```diff
@@ -14,19 +14,48 @@ describe('Crowdy News Topic twitter sharing', function () {
      * Set up before test methods.
      * Initiates all the page objects for testing.
      */
+    //Initialize the Browser instance before each test.
```

Comment:
This is not a line comment. There is already a block comment describing the event